kotest-runner/kotest-runner-junit5/src/jvmMain/kotlin/io/kotest/runner/junit/platform/KotestJunitPlatformTestEngine.kt

```diff
@@ -75,11 +75,19 @@ class KotestJunitPlatformTestEngine : TestEngine {
          .map { Class.forName(it).newInstance() as Extension }
          .forEach { configuration.registry.add(it) }
 
-      TestEngineLauncher(listener)
-         .withConfiguration(configuration)
-         .withExtensions(root.testFilters)
-         .withClasses(root.classes)
-         .launch()
+      if(root.isUniqueIdSelectorsRequest()) {
```

Comment:
We probably could get rid of this condition but I want the change to be minimal and only impact `discovery` with `UniqueIdSelectors`

Reply:
I got rid of it in a later commit.